### PR TITLE
feat: user guide zh

### PR DESCRIPTION
**Pull request by Aganxx**

### CODE REVIEW

File: docs/zh/user-guide/concepts/why-greptimedb.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+我们在设计和开发 GreptimeDB 时遵循以下原则：
+
+- **云原生**：从设计 GreptimeDB 的第一天起，其就被定位为一款完全利用云的基础设施和能力来运行和服务的云原生数据库，因此具有出色的可扩展性和容错性。
+- **用户友好**：我们明白开发者体验至关重要，GreptimeDB 的设计关键原则是对开发者友好。这不仅体现在开发、部署和运营方面，在与数据生态系统的无缝兼容性，以及不断完善的文档和社区指南方面，都无一不体现了 GreptimeDB 用户友好的特性。
+- **高性能**：处理时间序列数据的特性之一就是大规模的数据摄取、查询和分析。在高并发场景下依然持续进行性能优化也是我们设计 GreptimeDB 时的核心原则之一。
+- **灵活的架构**：通过抽象良好的分层和封装隔离，GreptimeDB 的部署形式可以满足从嵌入式、单机版、传统集群到云原生的各种环境。
+
+![](https://cdn.jsdelivr.net/gh/filess/img19@main/2023/06/02/1685689928655-5a1f1d0c-70cf-4df0-ba52-6c669062f8ee.png)
+
+GreptimeDB 的架构
```
Comment (bht991002) [inline docs/zh/user-guide/concepts/why-greptimedb.md:16]:
```suggestion
*GreptimeDB 的架构*
```

File: docs/zh/user-guide/concepts/why-greptimedb.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+我们的解决方案能更高效地管理海量的时间序列数据，并且能轻松处理时序数据的写入/查询和分析的混合负载需求。无论你的系统是在本地运行还是在云端，GreptimeDB 都可以轻松处理任何规模的负载。基于过去的经验，我们有信心为用户提供任何符合其需求的解决方案和服务。
+
+我们在设计和开发 GreptimeDB 时遵循以下原则：
+
+- **云原生**：从设计 GreptimeDB 的第一天起，其就被定位为一款完全利用云的基础设施和能力来运行和服务的云原生数据库，因此具有出色的可扩展性和容错性。
+- **用户友好**：我们明白开发者体验至关重要，GreptimeDB 的设计关键原则是对开发者友好。这不仅体现在开发、部署和运营方面，在与数据生态系统的无缝兼容性，以及不断完善的文档和社区指南方面，都无一不体现了 GreptimeDB 用户友好的特性。
+- **高性能**：处理时间序列数据的特性之一就是大规模的数据摄取、查询和分析。在高并发场景下依然持续进行性能优化也是我们设计 GreptimeDB 时的核心原则之一。
+- **灵活的架构**：通过抽象良好的分层和封装隔离，GreptimeDB 的部署形式可以满足从嵌入式、单机版、传统集群到云原生的各种环境。
+
+![](https://cdn.jsdelivr.net/gh/filess/img19@main/2023/06/02/1685689928655-5a1f1d0c-70cf-4df0-ba52-6c669062f8ee.png)
```
Comment (bht991002) [inline docs/zh/user-guide/concepts/why-greptimedb.md:14]:
```suggestion
<p><img src="/blogs/2023-06-01-why-greptimedb/image1.png" alt="The architecture of GreptimeDB" style="width: 70%; margin: 0 auto;" /></p>
```

File: docs/zh/user-guide/concepts/why-greptimedb.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+接下来，我们将逐一展开说明下这些设计原则。
+
+## 云原生
+
+GreptimeDB 专为云而生，充分利用云的优势，如弹性、可扩展性和高可用性。
+
+![](https://cdn.jsdelivr.net/gh/filess/img5@main/2023/06/02/1685690018913-600e3333-07b3-4bb3-a350-14e1dbdf8eab.png)
+
+存算分离/算算分离
```
Comment (bht991002) [inline docs/zh/user-guide/concepts/why-greptimedb.md:26]:
```suggestion
*存算分离/算算分离*
```

File: docs/zh/user-guide/concepts/why-greptimedb.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+GreptimeDB 的架构
+
+接下来，我们将逐一展开说明下这些设计原则。
+
+## 云原生
+
+GreptimeDB 专为云而生，充分利用云的优势，如弹性、可扩展性和高可用性。
+
+![](https://cdn.jsdelivr.net/gh/filess/img5@main/2023/06/02/1685690018913-600e3333-07b3-4bb3-a350-14e1dbdf8eab.png)
```
Comment (bht991002) [inline docs/zh/user-guide/concepts/why-greptimedb.md:24]:
```suggestion
<p><img src="/blogs/2023-06-01-why-greptimedb/image2.png" alt="Storage/Compute Disaggregation, Compute/Compute separation" style="width: 70%; margin: 0 auto;" /></p>
```

File: docs/zh/user-guide/concepts/why-greptimedb.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+- 在多个应用程序之间共享数据
+- 根据需求提供无限的并发扩展性
+
+## 用户友好
+
+### Time-Series Table，schemaless 设计
+
+「GreptimeDB」结合了 Metric (Measurement/Tag/Field/Timestamp) 模型和关系数据模型（表），提供了一个名为 Time-Series Table 的新数据模型，该模型以行和列的形式呈现数据，将 Metric 中的标签和字段映射到列上，同时通过 `Time Index` 约束来指定时间索引所在列。
+
+![](https://cdn.jsdelivr.net/gh/filess/img11@main/2023/06/02/1685690158466-e3d031a1-50d3-4ba9-9bb9-8a367b3e62bd.34)
```
Comment (bht991002) [inline docs/zh/user-guide/concepts/why-greptimedb.md:50]:
```suggestion
<p><img src="/blogs/2023-06-01-why-greptimedb/image3.png" alt="Time-Series Table" style="width: 70%; margin: 0 auto;" /></p>
```

File: docs/zh/user-guide/concepts/why-greptimedb.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+时间序列表格
```
Comment (bht991002) [inline docs/zh/user-guide/concepts/why-greptimedb.md:52]:
```suggestion
*时间序列表格*
```

File: docs/zh/user-guide/concepts/why-greptimedb.md (hunk 2 above)
Comment (Aganxx) [reply docs/zh/user-guide/concepts/why-greptimedb.md]: prompt: No path exists for "/blogs/2023-06-01-why-greptimedb/image1.png"

File: docs/zh/user-guide/concepts/why-greptimedb.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+
+## 灵活的架构
+
+通过模块化和分层设计，不同的模块和组件可以独立地开启、组合或分离。例如，我们可以将 Frontend、Datanode 和 Meta Server 合并为一个独立的二进制文件，并且我们也可以独立地为每个表启用或禁用 WAL。
+
+这种灵活的架构设计让 GreptimeDB 能在使用相同的 API 编程接口和控制面的条件下，满足从边缘到云各种场景中的部署和使用需求。
+
+## 小结
+
+随着时间序列数据的管理和分析需求不断增长，我们相信 GreptimeDB 能够提供一种高效、可扩展和用户友好的解决方案。我们将在 6 月初 推出分布式可用的版本 v0.3，无论你的应用场景是什么，我们都有信心为您提供满足需求的服务，也期待任何反馈和建议，以便我们不断改进我们的产品和服务。
```
Comment (bht991002) [inline docs/zh/user-guide/concepts/why-greptimedb.md:90]:
```suggestion
随着时间序列数据的管理和分析需求不断增长，我们相信 GreptimeDB 能够提供一种高效、可扩展和用户友好的解决方案。我们将在 6 月初推出分布式可用的版本 v0.3，无论你的应用场景是什么，我们都有信心为您提供满足需求的服务，也期待任何反馈和建议，以便我们不断改进我们的产品和服务。
```

File: docs/zh/user-guide/concepts/why-greptimedb.md [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+我们的解决方案能更高效地管理海量的时间序列数据，并且能轻松处理时序数据的写入/查询和分析的混合负载需求。无论你的系统是在本地运行还是在云端，GreptimeDB 都可以轻松处理任何规模的负载。基于过去的经验，我们有信心为用户提供任何符合其需求的解决方案和服务。
+
+我们在设计和开发 GreptimeDB 时遵循以下原则：
+
+- **云原生**：从设计 GreptimeDB 的第一天起，其就被定位为一款完全利用云的基础设施和能力来运行和服务的云原生数据库，因此具有出色的可扩展性和容错性。
+- **用户友好**：我们明白开发者体验至关重要，GreptimeDB 的设计关键原则是对开发者友好。这不仅体现在开发、部署和运营方面，在与数据生态系统的无缝兼容性，以及不断完善的文档和社区指南方面，都无一不体现了 GreptimeDB 用户友好的特性。
+- **高性能**：处理时间序列数据的特性之一就是大规模的数据摄取、查询和分析。在高并发场景下依然持续进行性能优化也是我们设计 GreptimeDB 时的核心原则之一。
+- **灵活的架构**：通过抽象良好的分层和封装隔离，GreptimeDB 的部署形式可以满足从嵌入式、单机版、传统集群到云原生的各种环境。
+
+<p><img src="/blogs/2023-06-01-why-greptimedb/image1.png" alt="The architecture of GreptimeDB" style="width: 70%; margin: 0 auto;" /></p>
```
Comment (bht991002) [inline docs/zh/user-guide/concepts/why-greptimedb.md:14]:
```suggestion
![Storage/Compute Disaggregation, Compute/Compute separation](../../../public/storage-compute-disaggregation-compute-compute-separation.png)
```

File: docs/zh/user-guide/concepts/why-greptimedb.md [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+
+*GreptimeDB 的架构*
+
+接下来，我们将逐一展开说明下这些设计原则。
+
+## 云原生
+
+GreptimeDB 专为云而生，充分利用云的优势，如弹性、可扩展性和高可用性。
+
+<p><img src="/blogs/2023-06-01-why-greptimedb/image2.png" alt="Storage/Compute Disaggregation, Compute/Compute separation" style="width: 70%; margin: 0 auto;" /></p>
```
Comment (bht991002) [inline docs/zh/user-guide/concepts/why-greptimedb.md:24]:
```suggestion
![Storage/Compute Disaggregation, Compute/Compute separation](../../../public/storage-compute-disaggregation-compute-compute-separation.png)
```

File: docs/zh/user-guide/concepts/why-greptimedb.md [hunk 10]
```diff
@@ -0,0 +1,90 @@
...
+- 在多个应用程序之间共享数据
+- 根据需求提供无限的并发扩展性
+
+## 用户友好
+
+### Time-Series Table，schemaless 设计
+
+「GreptimeDB」结合了 Metric (Measurement/Tag/Field/Timestamp) 模型和关系数据模型（表），提供了一个名为 Time-Series Table 的新数据模型，该模型以行和列的形式呈现数据，将 Metric 中的标签和字段映射到列上，同时通过 `Time Index` 约束来指定时间索引所在列。
+
+<p><img src="/blogs/2023-06-01-why-greptimedb/image3.png" alt="Time-Series Table" style="width: 70%; margin: 0 auto;" /></p>
```
Comment (bht991002) [inline docs/zh/user-guide/concepts/why-greptimedb.md:50]:
```suggestion
![Time-Series Table](../../../public/time-series-table.png)
```

**Pull request closed without merging**